### PR TITLE
(feat): dask tutorial

### PR DESCRIPTION
**Pull request by ilan-gold**
Ran into a little issue of exactly how to display this functionality because i/o from disk starting from a sparse representation is non-trivial (hence the various hacks) and _also_ the scanpy data is stored in a very old format so this does not lend itself well to writing a helpful/usable tutorial whose behavior people can mimic.....

### CODE REVIEW

**Comment by ivirshup** [comment]
@ilan-gold, did you ever end up trying the `slurm` schedueler? I'm not sure I want to recommend it if it's something we haven't tried.

**Comment by ilan-gold** [comment]
@ivirshup Fair point, we can remove it then.

**Comment by ilan-gold** [comment]
@ivirshup Why skip filtering now that the `obsp` is not being read in?

**Comment by ivirshup** [comment]
> Why skip filtering now that the obsp is not being read in?

It doesn't really need it mainly. Can try it.

----

I'm realizing we should probably prioritize dask support in `sc.get.aggregate` and then use that in the `sc.tl.rank_genes_groups`, as well as the dot-plot and heat map functions.

----

`noqa` comment added because of: https://github.com/astral-sh/ruff/issues/10357

File: dask.ipynb [hunk 1]
```diff
@@ -0,0 +1,942 @@
+{
```
Comment (ilan-gold) [inline dask.ipynb:1]: <p>Why commented out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scanpy-tutorials/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (ivirshup) [reply dask.ipynb:1]: <p>Since it takes about ~30 min to run, but good to note that this should be uncommented and run before finalization</p><p> </p>

Comment (ilan-gold) [reply dask.ipynb:1]: I'll add that it

**Pull request merged**